.mergify.yml

```diff
@@ -12,21 +12,18 @@ pull_request_rules:
   - status-success=f28-flake8
   - status-success=f28-pydocstyle
   - status-success=f28-python2-unit
```

Comment:
You can drop the unit tests too, no?

Reply:
We can't yet, because the client still supports Python 2.

Reply:
Python 2 support is planned to be dropped from the client in https://github.com/fedora-infra/bodhi/issues/2856. I am waiting just a bit to see if my dependencies will stop using it before making that change.